package.json

```diff
@@ -106,7 +106,7 @@
         "iconv-lite": "~0.6.3",
         "isomorphic-ws": "^5.0.0",
         "jsesc": "~3.0.2",
-        "jsonata": "^2.0.3",
+        "jsonata": "^2.0.4",
```

Comment:
[changelog](https://github.com/jsonata-js/jsonata/blob/master/CHANGELOG.md) looks fine for this one